manifests/4.8/cluster-secondary-scheduler-operator.v4.8.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,183 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  # The version value is substituted by the ART pipeline
+  name: secondaryscheduleroperator.v4.8.0
+  namespace: openshift-secondary-scheduler-operator
+  annotations:
+    alm-examples: |
+      [
+        {
+          "apiVersion": "operator.openshift.io/v1",
+          "kind": "SecondaryScheduler",
+          "metadata": {
+            "name": "cluster",
+            "namespace": "openshift-secondary-scheduler-operator"
+          },
+          "spec": {
+            "managementState": "Managed",
+            "schedulerConfig": "TargetLoadPacking"
+          }
+        }
+      ]
+    certified: "false"
+    containerImage: quay.io/openshift/secondary-scheduler-operator:4.8
+    createdAt: 2021/07/22
+    olm.skipRange: ">=4.6.0 <4.8.0"
+    description: An operator to run secondary-scheduler in Openshift cluster.
+    repository: https://github.com/openshift/secondary-scheduler-operator
+    support: Red Hat, Inc.
+    capabilities: Basic Install
+    categories: OpenShift Optional
+spec:
+  customresourcedefinitions:
+    owned:
+    - displayName: Secondary Scheduler
+      description: SecondaryScheduler is the Schema for the secondaryschedulers API
+      group: operator.openshift.io
+      kind: SecondaryScheduler
+      name: Secondaryschedulers.operator.openshift.io
+      version: v1
+  description: |
+      The Secondary Scheduler Operator provides the ability to a customized scheduler image developed using the scheduler plugin framework with customized configuration as a secondary scheduler in OpenShift.
+
+      ## Secondary Scheduler Configuration
+
+      Once the operator is installed, you can configure the plugins to run for your secondary scheduler.
+
+      There are several example secondary secondary schedulers you can run in your cluster:
+
+      * TargetLoadPacking: .
+      * .
+      * .
+
+      ## Additional Parameters
+
+      In addition to the profiles, the following parameters can be configured:
+
+      * `deschedulingIntervalSeconds` - Set the number of seconds between descheduler runs. A value of `0` in this field runs the descheduler once and exits.
+  displayName: Secondary Scheduler Operator
+  keywords: ["scheduling", "workload"]
+  provider:
+    name: Red Hat, Inc.
+  maturity: beta
+  version: 4.8.0
+  icon:
```

Comment:
we'll need to get an icon developed for this, I'm assuming this was just copied from descheduler :)